src/wallet/coinselection.h

```diff
@@ -109,6 +109,10 @@ class COutput
 
     bool m_from_me;
 
+    CAmount effective_value;
+    CAmount m_fee{0};
+    CAmount m_long_term_fee{0};
```

Comment:
864b4edf130afe2b77142ca343022b3de6d3fec5: this is not set and not used

Reply:
It is used later, but I didn't want that commit to be too big.

Reply:
06cf38fce16f327579b008ee8fb78fa851cc8fce: it compiles fine now. But ideally you would move some more code from d840d84f547e911ec88f098d1ece7ed86f100cbc into this commit to set and use them. Not sure if that's practical.